### PR TITLE
Add packages to start a package structure

### PR DESCRIPTION
**Pull request by llrs**
I added some packages related to starting a package. 
Besides the already listed, I added those that are on CRAN from the replies on [a poll](https://fosstodon.org/@jennybryan/111999826499572522).
I mixed the text describing what each package provides and the name & link of the packages.

### CODE REVIEW

**Comment by llrs** [comment]
I made this small PR to be reviewed, @hturner and @rsbivand. 
I don't know if you prefer constant small PR or a big one (I have made a big modification to the proposal on my machine). 

I would also like to ask if I should add the header as in other CTV or this will be added later as per policy (I have a separate branch for that too if needed). 

Let me know how you prefer to work on this.

**Comment by llrs** [comment]
Hi, any feedback on the last comment and this PR? Ideally I would like for this CTV to be officially included before the useR!2024 conference.

File: proposal.md [hunk 1]
```diff
@@ -68,14 +69,16 @@ code files provided to appropriate places, and creates skeleton help
...
-Structure](https://cran.r-project.org/doc/manuals/r-release/R-exts.html#Package-structure).
+The `r pkg("usethis", priority = "core")` package has many utilities to setup the structure of the package, including a NEWS file and adding dependencies in the README.
+The `r pkg("pkgKitten")` package provides an almost empty package skeleton and some utilities to create help pages.
+The `r bioc("biocthis")` package prepares for Bioconductor repository.
 
-CONSIDER: usethis, biocthis, fusen, litr, noweb, rfold, many, DataPackageR
+Some packages like `r pkg("fusen")` and `r github("jacobbien/litr-project")` prepare a project from an R markdown file. 
+`r pkg("noweb")` allows to use literate programming to create the package.
+There are others that help packaging several R files, `r pkg("rfold")`, or create a package from a dataset `r pkg("DataPackageR")`.
+Others, like `r pkg("cookiecutter")` can create a new package from a template stored elsewhere.
```
Comment (hturner) [inline proposal.md:81]: Please use bullet points for the contributed packages/services, so the overall structure of each section is 

<opening comments, highlighting base R/official R project tools>
<list of contributed packages/services>

If necessary, you can use sectioning to provide further structure. 

Although existing task views do often have lists of the form

* The [foo]() package does this
* The [bar]() package does that

I think it is better to simplify to

* [foo]() does this
* [bar]() does that

unless you are talking about something that is not an R package.

**Review by hturner — COMMENTED** [review]
Sorry to be so slow in reviewing this PR. The packages you have selected and the comments you have made are fine, I just have a comment on the structure.

I'm happy to look at a big PR that follows this structure

**Pull request merged**